### DISCUSSION TITLE
svp4pro使用rife插帧3倍4k视频可以流畅本项目却会卡顿？

### DISCUSSION THREAD

**Discussion by paynetsang** [hooke007/mpv_PlayKit#305] [回收]
看了一下模型应该都是一样的，是哪里占用的问题吗？另外默认的解码器效果比d3d11va更好？如果要更改的话是改成hwdec=d3d11va就行了是吧老哥？怎么实现每次启动自动启用rife插帧呢？

**Comment by hooke007** [comment]
1. 解码无任何效果区别
2. 没去研究过脚本细节，但是我预测是降低分辨率的小把戏，跟我在rife_2x里做的差不多。你没说你的显卡是什么，在4k3x的无质量损失的条件下，4080是不够的
3. 参见mpv.conf里说vf说明

**Reply by paynetsang** [reply]
谢谢老哥，我是3070tilaptop，rife设置里面ensemble版模型有什么不同吗？显卡线程数应该填多少呢，实在没找到参考

**Reply by hooke007** [reply]
tta有好有坏，更吃性能；线程没必要改

**Reply by paynetsang** [reply]
好的，谢谢
